### PR TITLE
CssTask 3

### PR DESCRIPTION
**Pull request by dudzon**
https://dudzon.github.io/ui-internship-lwo2/css_tasks/task3/

### CODE REVIEW

File: css_tasks/task3/main.css [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    font-size:2rem;
+    width: 92%;
+    display:flex;
+    margin: 0 0 0 .7rem
+}
+.line--sixth{
+    margin:.5rem 0 0 .8rem;
+    font-size:1.5rem;
+    display:flex;
+    width:90%;
```
Comment (tsvetkovv) [inline css_tasks/task3/main.css:44]: why `width`? and horizontal margins try to mitigate this..

File: css_tasks/task3/main.css [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+.eye-chart{
+    width:20rem;
+    height:25.5rem;
+    margin:0 auto;
+    background:#FDFCF8;
+}
+.eye-chart__line{
+    display:flex;
+    width:95%;
+    margin:0 auto;
```
Comment (tsvetkovv) [inline css_tasks/task3/main.css:17]: try to disable this line in dev tools.. something changed?

File: css_tasks/task3/main.css [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    width: 92%;
+    display:flex;
+    margin: 0 0 0 .7rem
+}
+.line--sixth{
+    margin:.5rem 0 0 .8rem;
+    font-size:1.5rem;
+    display:flex;
+    width:90%;
+}
```
Comment (tsvetkovv) [inline css_tasks/task3/main.css:45]: ideally, lines differ from each other in size

File: css_tasks/task3/main.css [hunk 4]
```diff
@@ -0,0 +1,45 @@
+*{
+    margin:0;
```
Comment (tkachukivan) [inline css_tasks/task3/main.css:2]: generally it is a bit easier to read when you have space after selector and space before property value

```
selector {
  margin: 0;
}
```

**Review by tkachukivan — APPROVED** [review]

File: css_tasks/task3/main.css (hunk 4 above)
Comment (tkachukivan) [reply css_tasks/task3/main.css:2]: Also always try to follow some properties order when write styles, you need to have some logic behind why your properties in some order [link](https://webdesign.tutsplus.com/articles/outside-in-ordering-css-properties-by-importance--cms-21685)

**Pull request merged**